share/p2p/shrexeds/client.go

```diff
@@ -53,6 +54,9 @@ func (c *Client) RequestEDS(
 	if err == nil {
 		return eds, nil
 	}
+	if err != nil {
+		log.Debugw("client: eds request to peer failed", "peer", peer, "hash", dataHash.String(), "error", err)
+	}
```

Comment:
Nit: we know error is not nil already by if above

Reply:
good catch, fixed